scripts/figma-to-css.mjs

```diff
@@ -97,6 +97,7 @@ const figmaToCss = async () => {
         case 'Frame 1105':
           if (frame?.children.length > 1) {
             return frame.children
+              .reverse()
               ?.reduce((acc, { children }) => acc.concat(children), [])
               ?.filter(({ characters }) => !characters?.match(/Desktop|Mobile/g))
               ?.map(getFontProperties);
```

Comment:
```suggestion
              ?.map(getFontProperties)
              ?.reverse();
```
WDYT? IMO .reverse at last operation looks better (for me) 
WHY? When I get results without .reverse I have "right" result but I need to reverse it to adjust Desktop -> Mobile order to Mobile->Destkop

Reply:
changed